Assets/MixedRealityToolkit/_Core/Definitions/Devices/ControllerMappingLibrary.cs

```diff
@@ -224,10 +225,588 @@ public static class ControllerMappingLibrary
+                }
+
+                return touchControllerLeftBlackScaled;
+            }
+        }
+
+        private static Texture2D touchControllerRightWhite;
+
+        public static Texture2D TouchControllerRightWhite
+        {
+            get
+            {
+                if (touchControllerRightWhite == null)
+                {
+                    touchControllerRightWhite = (Texture2D)AssetDatabase.LoadAssetAtPath("Assets/MixedRealityToolkit/_Core/Resources/Textures/OculusControllersTouch_right_white.png", typeof(Texture2D));
+                }
+
+                return touchControllerRightWhite;
+            }
+        }
+
+        private static Texture2D touchControllerRightWhiteScaled;
+
+        public static Texture2D TouchControllerRightWhiteScaled
+        {
+            get
+            {
+                if (touchControllerRightWhiteScaled == null)
+                {
+                    touchControllerRightWhiteScaled = (Texture2D)AssetDatabase.LoadAssetAtPath("Assets/MixedRealityToolkit/_Core/Resources/Textures/OculusControllersTouch_right_white_scaled.png", typeof(Texture2D));
+                }
+
+                return touchControllerRightWhiteScaled;
+            }
+        }
+
+        private static Texture2D touchControllerRightBlack;
+
+        public static Texture2D TouchControllerRightBlack
+        {
+            get
+            {
+                if (touchControllerRightBlack == null)
+                {
+                    touchControllerRightBlack = (Texture2D)AssetDatabase.LoadAssetAtPath("Assets/MixedRealityToolkit/_Core/Resources/Textures/OculusControllersTouch_right_black.png", typeof(Texture2D));
+                }
+
+                return touchControllerRightBlack;
+            }
+        }
+
+        private static Texture2D touchControllerRightBlackScaled;
+
+        public static Texture2D TouchControllerRightBlackScaled
+        {
+            get
+            {
+                if (touchControllerRightBlackScaled == null)
+                {
+                    touchControllerRightBlackScaled = (Texture2D)AssetDatabase.LoadAssetAtPath("Assets/MixedRealityToolkit/_Core/Resources/Textures/OculusControllersTouch_right_black_scaled.png", typeof(Texture2D));
+                }
+
+                return touchControllerRightBlackScaled;
+            }
+        }
+
+        private static Texture2D viveWandControllerLeftWhite;
+
+        public static Texture2D ViveWandControllerLeftWhite
+        {
+            get
+            {
+                if (viveWandControllerLeftWhite == null)
+                {
+                    viveWandControllerLeftWhite = (Texture2D)AssetDatabase.LoadAssetAtPath("Assets/MixedRealityToolkit/_Core/Resources/Textures/ViveWandController_left_white.png", typeof(Texture2D));
```

Comment:
I would comment / remove this code (entire vive block) while awaiting guidance on images